### PR TITLE
DAH-1392: SF.gov link paths

### PR DESCRIPTION
**Pull request by cliu02**
[DAH-1392](https://sfgovdt.jira.com/browse/DAH-1392)

[DAH-1392]: https://sfgovdt.jira.com/browse/DAH-1392?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/javascript/util/languageUtil.tsx [hunk 1]
```diff
@@ -141,6 +141,23 @@ export const getCurrentLanguage = (path?: string | undefined): LanguagePrefix =>
...
+      return `https://sf.gov/es/node/${node}`
+    case "tl":
+      return `https://sf.gov/fil/node/${node}`
+    case "zh":
+      return `https://sf.gov/zh-hant/node/${node}`
+    default:
+      return enLink
+  }
+}
+
```
Comment (alulabeshue-sfgov) [inline app/javascript/util/languageUtil.tsx:160]: I think we can add some unit tests for this function.

File: app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx [hunk 2]
```diff
@@ -147,7 +152,10 @@ export const ListingDetailsEligibility = ({
                   <p>
                     {renderInlineMarkup(
                       t("listings.customListingType.educator.eligibility.part2", {
-                        chisholmLink: "https://sf.gov/apply-shirley-chisholm-village-housing",
+                        chisholmLink: getSfGovUrl(
+                          "https://sf.gov/apply-shirley-chisholm-village-housing",
+                          10543
+                        ),
```
Comment (alulabeshue-sfgov) [inline app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx:158]: [Below](https://github.com/SFDigitalServices/sf-dahlia-web/blob/DAH-1392-sfgov-links/app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx#L204) there is another reference to this same `chisholmLink`

**Review by alulabeshue-sfgov — COMMENTED** [review]
Thanks for looking into this, looks good so far. I left few comments. 

I also noticed that there are links to sf.gov in [additional-resources](https://github.com/SFDigitalServices/sf-dahlia-web/blob/DAH-1392-sfgov-links/app/assets/json/additional-resources.json) and in the [translation files](https://github.com/SFDigitalServices/sf-dahlia-web/blob/DAH-1392-sfgov-links/app/assets/json/translations/react/en.json). Do those links need to go to the corresponding language as well?

**Review by alulabeshue-sfgov — APPROVED** [review]

**Comment by jimlin-sfgov** [comment]
@cliu02 It looks like translation values were modified. There is this ticket related to modifying translation values: https://sfgovdt.jira.com/browse/DAH-1746

>  ...changed English strings with the assumption that Phrase would detect those changed strings and make them unverified, or “fuzzy”. Phrase did not do this, so when the English string changed, the other languages did not change with it.

Is this still an issue, and/or is there something we are doing to handle this?

**Pull request merged**